adapters/bidder.go

```diff
@@ -0,0 +1,69 @@
+package adapters
```

Comment:
Open question I'm seeking opinions on: should we rename the `adapters` package to `bidders`?

Reply:
One thing to consider is the infrastructure backing things. Like DistrictM is hosted on AppNexus, They are considered a different "bidder" but would still use the AppNexus "adapter"

Reply:
I think I prefer the existing naming convention. I can see eventually moving to OpenRTB becoming the primary or only endpoint. If primary we would have to consider moving all the legacy code into a legacy/ directory or some other name, to keep things consistent. If we drop legacy all together, then everything is in openrtb2. Might not be bad if we eventually add another endpoint, but otherwise we are left with an unneeded directory in the middle of things.